src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java

```diff
@@ -343,26 +343,37 @@ public int getProtocolVersion() {
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Teleport Entities: " + this.isTeleportingEntities());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Send Disabled Portal Message: " + this.isSendingDisabledPortalMessage());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Send No Destination Message: " + this.isSendingNoDestinationMessage());
         buffer += logAndAddToPasteBinBuffer("Special Code: FRN001");
         return buffer;
     }
 
     private String logAndAddToPasteBinBuffer(String string) {
         this.log(Level.INFO, string);
-        return "[Multiverse-NetherPortals] " + string + "\n";
+        return "[Multiverse-NetherPortals] " + string + System.lineSeparator();
     }
 
     public String getVersionInfo() {
-        StringBuilder buffer = new StringBuilder();
-        buffer.append("[Multiverse-NetherPortals] Multiverse-NetherPortals Version: ").append(this.getDescription().getVersion()).append('\n');
-        buffer.append("[Multiverse-NetherPortals] World links: ").append(this.getWorldLinks()).append('\n');
-        buffer.append("[Multiverse-NetherPortals] Nether Prefix: ").append(netherPrefix).append('\n');
-        buffer.append("[Multiverse-NetherPortals] Nether Suffix: ").append(netherSuffix).append('\n');
-        buffer.append("[Multiverse-NetherPortals] Special Code: ").append("FRN001").append('\n');
-        return buffer.toString();
+        return "[Multiverse-NetherPortals] Multiverse-NetherPortals Version: " + this.getDescription().getVersion() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Nether Prefix: " + this.getNetherPrefix() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Nether Suffix: " + this.getNetherSuffix() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] End Prefix: " + this.getEndPrefix() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] End Suffix: " + this.getEndSuffix() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Nether Links: " + this.getWorldLinks() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] End Links: " + this.getEndWorldLinks() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Bounceback: " + this.isUsingBounceBack() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Teleport Entities: " + this.isTeleportingEntities() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Send Disabled Portal Message: " + this.isSendingDisabledPortalMessage() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Send No Destination Message: " + this.isSendingNoDestinationMessage() + System.lineSeparator()
+                + "[Multiverse-NetherPortals] Special Code: " + "FRN001" + System.lineSeparator();
     }
 }
```

Comment:
Where is this used?

Reply:
It's used in the MVVersionEvent listener.